src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
 // write your code here
+
+const element = document.querySelectorAll('.population');
```

Comment:
```suggestion
const population = [...document.querySelectorAll('.population')];
```

It will be an array, so it will allow you to get rid of `count`